src/hotspot/share/gc/g1/g1CardSet.hpp

```diff
@@ -194,23 +191,23 @@ class G1CardSet : public CHeapObj<mtGCCardSet> {
   static G1CardSetCoarsenStats _coarsen_stats; // Coarsening statistics since VM start.
   static G1CardSetCoarsenStats _last_coarsen_stats; // Coarsening statistics at last GC.
 public:
-  // Two lower bits are used to encode the card storage types
-  static const uintptr_t CardSetPtrHeaderSize = 2;
+  // Two lower bits are used to encode the card set container types
+  static const uintptr_t ContainerPtrHeaderSize = 2;
 
-  // CardSetPtr represents the card storage type of a given covered area. It encodes
-  // a type in the LSBs, in addition to having a few significant values.
+  // ContainerPtr represents the card set container  type of a given covered area.
+  // It encodes a type in the LSBs, in addition to having a few significant values.
   //
   // Possible encodings:
   //
   // 0...00000 free               (Empty, should never happen)
-  // 1...11111 full               All card indexes in the whole area this CardSetPtr covers are part of this container.
-  // X...XXX00 inline-ptr-cards   A handful of card indexes covered by this CardSetPtr are encoded within the CardSetPtr.
+  // 1...11111 full               All card indexes in the whole area this ContainerPtr covers are part of this container.
+  // X...XXX00 inline-ptr-cards   A handful of card indexes covered by this ContainerPtr are encoded within the ContainerPtr.
   // X...XXX01 array of cards     The container is a contiguous array of card indexes.
   // X...XXX10 bitmap             The container uses a bitmap to determine whether a given index is part of this set.
-  // X...XXX11 howl               This is a card set container containing an array of CardSetPtr, with each CardSetPtr
+  // X...XXX11 howl               This is a card set container containing an array of ContainerPtr, with each ContainerPtr
   //                              limited to a sub-range of the original range. Currently only one level of this
   //                              container is supported.
-  typedef void* CardSetPtr;
+  using ContainerPtr = void*;
```

Comment:
I can't anchor this elsewhere: the below `CardSetInlinePtr` etc tag names should also be `ContainerInlinePtr` etc, shouldn't they?